### DISCUSSION TITLE
is winget updated on its own?

### DISCUSSION THREAD

**Discussion by nazar2sfive** [microsoft/winget-cli#1101] [Q&A] [answered]
currently its v1.0.11451

when there is next release how does update happens?? AUTOmatically?

**Comment by denelon** [comment, marked as answer]
If you are on "[Windows Insider](https://insider.windows.com/) DEV" build, you will automatically get the next **development** release.

If you are signed into the Microsoft Store Account with an MSA that has been added to the "Windows Package Manager Insider" program you will automatically get the next **development** release. You can [sign up](http://aka.ms/winget-InsiderProgram) to become an insider. The process is manual and may take a few days after you receive the e-mail notification.

Once we push the "App Installer" with the "Windows Package Manager" to Windows 10 builds 1809 and newer via the Microsoft Store (September 21' timeframe *not a commitment*), **stable** releases are expected to follow (automatically) roughly quarterly. The delay is intentional to allow enterprise customers enough time to implement and deploy appropriate [Group Policy](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/manage-windows-package-manager-with-group-policy/ba-p/2346322) configurations.

All three of the scenarios above are true even if you installed the "App Installer" from the "Windows Package Manager" releases via GitHub. You may install one of the [releases](https://github.com/microsoft/winget-cli/releases) from GitHub any time. Once a newer version has been published to the Microsoft Store, you will receive that update unless you or your enterprise has disabled updates from the Microsoft Store.
